docs/settings.md

```diff
@@ -33,10 +33,17 @@ $ pip install uvicorn[watchgodreload]
 
 ## Logging
 
-* `--log-config <path>` - Logging configuration file.
+* `--log-config <path>` - Logging configuration file. **Options:** *`dictConfig()` formats: .json, .yaml*. Any other format will be processed with `fileConfig()`. Set the `formatters.default.use_colors` and `formatters.access.use_colors` values to override the auto-detected behavior.
```

Comment:
are there any other format than ini that can be processed by `fileConfig()` ?

Reply:
There is only one file format that `fileConfig()` supports and that's the `configparser` [format](https://docs.python.org/3/library/logging.config.html#logging-config-fileformat). It's not precisely `.ini`, but `.ini` is the closest well-known format that we can use to give a hint as to what one should use if going the `fileConfig()` route. These changes don't require the `.ini` extension, it's just the fallback if `.json` and `.yaml` are not detected.

I would expect anyone wishing to use `fileConfig()` would be acquainted well-enough with it's format that seeing `.ini` wouldn't cause any confusion.